src/test/java/org/apache/mesos/reconciliation/DefaultReconcilerTest.java

```diff
@@ -192,6 +192,32 @@ public void testForceCompleteReconciler() {
         verifyZeroInteractions(mockDriver);
     }
 
+    @Test
+    public void testTaskLostToTaskRunningTransition() {
+        reconciler.start(Arrays.asList(TASK_STATUS_2));
+
+        assertFalse(reconciler.isReconciled());
+        assertEquals(1, reconciler.remaining().size());
+
+        reconciler.reconcile(mockDriver);
```

Comment:
This call is supposed to do nothing right?  We should validate it does nothing or remove the call.